pandas/tests/io/excel/test_readers.py

```diff
@@ -571,7 +572,7 @@ def test_reader_dtype_str(self, read_ext, dtype, expected):
         actual = pd.read_excel(basename + read_ext, dtype=dtype)
         tm.assert_frame_equal(actual, expected)
 
-    def test_dtype_backend(self, read_ext, dtype_backend):
+    def test_dtype_backend(self, request, engine, read_ext, dtype_backend):
```

Comment:
I don't think you need to change the paramters for functions that don't use the request / engine fixtures?

Reply:
Oops, sorry, it is from an old version PR. Fixed.